contrib/mesos/docs/issues.md

```diff
@@ -1,34 +1,118 @@
 ## Known Issues
 
-### Pod Placement
+This page identifies significant known issues with the Kubernetes-Mesos distribution.
+
+* [General Known Issues](#general-known-issues)
+* [DCOS Package Known Issues](#dcos-package-known-issues), in addendum to the above.
+
+## General Known Issues
+
+These known issues apply to all builds of Kubernetes-Mesos.
+
+### Upgrades
+
+Upgrading your Kubernetes-Mesos cluster is currently "YMMV" status.
```

Comment:
What is a "YMMV" status? Even when knowing this acronym I'd still be puzzled ;-) I'd be very explicit here, i.e.
"Upgrading a Kubernetes-Mesos cluster is currently unsupported."